src/main/java/club/minnced/discord/webhook/WebhookClient.java

```diff
@@ -25,25 +25,31 @@
 import club.minnced.discord.webhook.send.WebhookMessageBuilder;
 import club.minnced.discord.webhook.util.ThreadPools;
 import club.minnced.discord.webhook.util.WebhookErrorHandler;
+
 import okhttp3.OkHttpClient;
 import okhttp3.RequestBody;
 import okhttp3.Response;
+
+import org.checkerframework.checker.index.qual.NonNegative;
+
 import org.jetbrains.annotations.Async;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
+
 import org.json.JSONException;
 import org.json.JSONObject;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.annotation.Nonnegative;
```

Comment:
Don't just change all the annotations for no reason please

Reply:
Sorry, my IDE wasn't compiling cause they didn't exist. I added the dependency to the build file as I tried adding locally but that didn't work either.